providers/git/docs/changelog.rst

```diff
@@ -15,6 +15,25 @@
     specific language governing permissions and limitations
     under the License.
 
+0.0.4
```

Comment:
Would it now be the time to make a 1.0.0 of the provider? Or shall we still keep it with this low version number?